mon-pix/app/components/scorecard-details.hbs

```diff
@@ -86,9 +86,9 @@
             @isBorderVisible={{true}}
             @shape="rounded"
             @triggerAction={{this.openModal}}
+            aria-label="{{t 'pages.competence-details.for-competence' competence=@scorecard.name}}"
```

Comment:
Cette traduction ne peut pas fonctionner seule dans un aria-label car le texte qu'elle apporte vient en complément du reste du label du bouton : `"for-competence": "la compétence {competence}",`.

Pour corriger ce problème d'accessibilité, il est possible de transformer la `div` en un `span`